docs/interfaces/autoupdate-service.txt

```diff
@@ -85,10 +81,41 @@ Interface GenericModelDecriptor {
  * - If this interface is used in a field indication a relation, the id(s) are
  *   given by the actual model data.
  */
-Interface ModelDescriptor extends GenericModelDescriptor {
+Interface RelationFieldDescriptor extends Fields {
+    type: 'relation' | 'relation-list';
     collection: Collection;
 }
 
+/**
+ * Structured Fields: A field with `$` as a placeholder can be structured. The
+ * `$` is a template placeholder. The `template field` holds a list of strings
+ * with options to insert into `$`. All these fields are the `structured
+ * fields`. Example:
+ * B_$_ids is the remplate field. Its value may be `["1", "test"]`, so the
+ * structured fields are B_1_ids and B_test_ids.
+ *
+ * This interface indicates, that there are structured fields related to the
+ * given template field, that should be resolved. If one would give `B_$_ids:
+ * null` without using this interface, the response would be just the template
+ * field `B_$_ids` and no structured fields.
+ *
+ * For just resolving values, specify `values: null`.
```

Comment:
The attribute `values` is not needed at all. But `values: null` also works.

Reply:
Ah ok, this wasn't clear from your email